arch/arm/soc/nxp_lpc/lpc54xxx/soc.h

```diff
@@ -23,6 +23,17 @@ extern "C" {
 #include <device.h>
 #include <misc/util.h>
 #include <fsl_common.h>
+
+/* Address of RAM, where the image for core1 should be copied */
```

Comment:
The addresses seem like the should be board specific.

Reply:
The address should be shared by two boards in fact: the M4 version and M0 version. But these are separate boards in Zephyr now, so unfortunately there will stay some sort of duplication. But for sure it can be done at least better than now.